api/prisma/schema.prisma

```diff
@@ -2,24 +2,18 @@
 // learn more about it in the docs: https://pris.ly/d/prisma-schema
 
 generator client {
-  provider        = "prisma-client-js"
-  previewFeatures = ["nativeTypes"]
+  provider = "prisma-client-js"
 }
 
 datasource db {
-  provider = "sqlite"
+  provider = "mongodb"
   url      = env("DATABASE_URL")
 }
 
 model User {
-  id    Int    @id @default(autoincrement())
+  id String @id @default(auto()) @map("_id") @db.ObjectId
```

Comment:
Why is the id a string?

Reply:
See official Prisma and MongoDB Doc: 

> For the User model, we'll need to update the id. Instead of an Int, we'll use String. We'll set the default to auto(). Since MongoDB names the id field _id, we'll map the id field to _id. Lastly, we'll tell Prisma to use the data type of ObjectId for the id field.

https://www.mongodb.com/developer/how-to/type-safety-with-prisma-and-mongodb/